lib/src/validator/size.dart

```diff
@@ -21,8 +24,21 @@ class SizeValidator extends Validator {
     return packageSize.then((size) {
       if (size <= _MAX_SIZE) return;
       var sizeInMb = (size / math.pow(2, 20)).toStringAsPrecision(4);
-      errors.add("Your package is $sizeInMb MB. Hosted packages must be "
-          "smaller than 100 MB.");
+      // Current implementation of Package.listFiles skips hidden files
+      var ignoreExists = fileExists(join(entrypoint.root.dir, '.gitignore'));
```

Comment:
You can write the path as `entrypoint.root.path('.gitignore')`.

Reply:
Will do.